packages/neutrino-preset-jest/src/index.js

```diff
@@ -6,13 +6,25 @@ const { tmpdir } = require('os');
 const clone = require('lodash.clonedeep');
 const loaderMerge = require('neutrino-middleware-loader-merge');
 
+function getFinalPath(path) {
+  if (path[0] === '/') {
```

Comment:
OK, I think I get it now. So to make this a little more portable, let's bring in `isAbsolute` from path and try this:

```js
const getFinalPath = path => {
  if (isAbsolute(path)) {
    return path;
  }

  return path.startsWith('.') ?
    join('<rootDir>', path) :
    join('<rootDir>', 'node_modules', path);
};
```